docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -366,8 +366,56 @@ revisit this and keep track of dad as well as moms).
 Code for pulling GBD ASFR appears in `recent Maternal IV Iron model
 <https://github.com/ihmeuw/vivarium_gates_iv_iron/blob/67bbb175ee42dce4536092d2623ee4d83b15b080/src/vivarium_gates_iv_iron/data/loader.py#L166>`_.
 
-Multiparity --- make twins with probability from ACS, about 2.5%.  See
-Section (16) for additional details.
+Multiparity --- make twins with probability 4%.  See Section (16) for
+additional details.
+
+Relationship -- the sim knows a parent-child dyad when the new
+simulant is initialized, and to come up with a consistent value for
+the "reference person" relationship field, we use the following
+mapping:
+
++--------------------------------------------------+----------------------------------------+
+| Parent relationship to reference person          | Child relationship to reference person |
++==================================================+========================================+
+| Reference Person                                 | Biological child                       |
++--------------------------------------------------+----------------------------------------+
+| Opposite-sex husband/wife/spouse                 | Biological child                       |
```

Comment:
Is there no situation where we initialize simulants into any of the non-biological child relationships? I expect simulants aren't transitioning into those relationships either, so they will get smaller as the sim goes on.

Maybe a nice-to-have would be some probability of each of the child relationships. May depend a bit on the sex question I asked above.